cedar-policy-core/src/est/expr.rs

```diff
@@ -1302,12 +1302,14 @@ impl TryFrom<cst::Literal> for Expr {
             cst::Literal::False => Ok(Expr::lit(JSONValue::Bool(false))),
             cst::Literal::Num(n) => {
                 Ok(Expr::lit(JSONValue::Long(n.try_into().map_err(|_| {
-                    ParseError::ToAST("numeric literal out of range".to_string())
+                    ParseError::ToAST(format!("numeric literal out of range: {n}"))
                 })?)))
             }
             cst::Literal::Str(ASTNode { node, .. }) => match node {
                 Some(cst::Str::String(s)) => Ok(Expr::lit(JSONValue::String(s))),
-                Some(_) => Err(ParseError::ToAST("invalid string".to_string()).into()),
+                Some(cst::Str::Invalid(s)) => {
+                    Err(ParseError::ToAST(format!("invalid string: {s}")).into())
```

Comment:
might be a good idea to use `{s:?}` instead of `{s}` in this message, since I'm imagining invalid strings might contain weird characters or whitespace or etc